packages/office-ui-fabric-react/src/components/SwatchColorPicker/SwatchColorPicker.Props.ts

```diff
@@ -105,20 +106,7 @@ export interface ISwatchColorPickerItemProps {
   index?: number;
 
   /**
-   * The menu item button props. This value is only used if
-   * the type is MenuItem
-   */
-  menuItemButtonProps?: IButtonProps;
-
-  /**
    * Is this individual item disabled?
    */
   disabled?: boolean;
```

Comment:
I don't see why not, although I'm not aware of any instance where this happens currently in our implementation... I'll remove it and it could always be added back if we find a case where it's needed